packages/contracts/contracts/L1/messaging/L1StandardBridge.sol

```diff
@@ -0,0 +1,289 @@
+     * Constructor *
+     ***************/
+
+    // This contract lives behind a proxy, so the constructor parameters will go unused.
+    constructor() {}
+
+    /******************
+     * Initialization *
+     ******************/
+
+    /**
+     * @param _optimismPortal OptimismPortal address.
+     * @param _l2TokenBridge L2 standard bridge address.
+     */
+    // slither-disable-next-line external-function
+    function initialize(OptimismPortal _optimismPortal, address _l2TokenBridge) public {
+        require(address(optimismPortal) == address(0), "Contract has already been initialized.");
+        optimismPortal = _optimismPortal;
+        l2TokenBridge = _l2TokenBridge;
+    }
+
+    /**************
+     * Depositing *
+     **************/
+
+    /** @dev Modifier requiring sender to be EOA. This check could be bypassed by a malicious
+     *  contract via initcode, but it takes care of the user error we want to avoid.
+     */
+    modifier onlyEOA() {
+        // Used to stop deposits from contracts (avoid accidentally lost tokens)
+        require(!Address.isContract(msg.sender), "Account not EOA");
+        _;
+    }
+
+    /**
+     * @dev This function can be called with no data
+     * to deposit an amount of ETH to the caller's balance on L2.
+     * Since the receive function doesn't take data, a conservative
+     * default amount is forwarded to L2.
+     */
+    receive() external payable onlyEOA {
+        _initiateETHDeposit(msg.sender, msg.sender, 200_000, bytes(""));
+    }
+
+    /**
+     * @inheritdoc IL1StandardBridge
+     */
+    function depositETH(uint32 _l2Gas, bytes calldata _data) external payable onlyEOA {
+        _initiateETHDeposit(msg.sender, msg.sender, _l2Gas, _data);
+    }
+
+    /**
+     * @inheritdoc IL1StandardBridge
+     */
+    function depositETHTo(
+        address _to,
+        uint32 _l2Gas,
+        bytes calldata _data
+    ) external payable {
+        _initiateETHDeposit(msg.sender, _to, _l2Gas, _data);
+    }
+
+    /**
+     * @dev Performs the logic for deposits by storing the ETH and informing the L2 ETH Gateway of
+     * the deposit.
+     * @param _from Account to pull the deposit from on L1.
+     * @param _to Account to give the deposit to on L2.
+     * @param _l2Gas Gas limit required to complete the deposit on L2.
+     * @param _data Optional data to forward to L2. This data is provided
+     *        solely as a convenience for external contracts. Aside from enforcing a maximum
+     *        length, these contracts provide no guarantees about its content.
+     */
+    function _initiateETHDeposit(
+        address _from,
+        address _to,
+        uint32 _l2Gas,
+        bytes memory _data
+    ) internal {
+        // Construct calldata for finalizeDeposit call
+        bytes memory message = abi.encodeWithSelector(
+            IL2ERC20Bridge.finalizeDeposit.selector,
+            address(0),
+            Lib_PredeployAddresses.OVM_ETH,
```

Comment:
I believe that this is incorrect, because `OVM_ETH` is going to be removed from bedrock